applications/tari_swarm_daemon/docker/README.md

```diff
@@ -8,8 +8,9 @@ cd sources
 git clone https://github.com/tari-project/tari.git
 git clone https://github.com/tari-project/tari-dan.git
 git clone https://github.com/tari-project/tari-connector.git
-cp -v applications/tari_swarm/docker/cross-compile-aarch64.sh .
-docker build -f applications/tari_swarm/docker/tari_swarm.Dockerfile \
+cp -v tari-dan/applications/tari_swarm_daemon/docker/cross-compile-aarch64.sh .
+cd ..
+docker build -f sources/tari-dan/applications/tari_swarm_daemon/docker/tari_swarm.Dockerfile \ 
```

Comment:
Would this work too? Haven't tested but I think an escape newline is missing 

```suggestion
docker build -f tari-dan/applications/tari_swarm_daemon/docker/tari_swarm.Dockerfile \ 
```

Reply:
building inside `sources` dir won't work as `tari_swarm.Dockerfile` has at line 151
```dockerfile
ADD sources/tari-dan /home/tari/sources/tari-dan
```
So it's assumes we are building in a parent directory of `sources` dir

Reply:
Another way would be to change Dockerfile but it would be more work than simple `cd ..`

Reply:
Ah thanks, I was reviewing on my phone so didn't see the whole context